### PR TITLE
Create atlier.js to to process api request for products, reviews. QA,…

### PR DESCRIPTION
**Pull request by mangohead33**
… Cart, and other request can be added in the future. Update index so that it can porcess said request

### CODE REVIEW

File: server/atlier.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
+const axios = require('axios');
```
Comment (juliebarwick) [inline server/atlier.js]: Suggest rename file to atelier.js

File: server/index.js [hunk 2]
```diff
@@ -1,11 +1,85 @@
 const express = require('express');
+const path = require('path');
+const atlier = require('./atlier');
```
Comment (juliebarwick) [inline server/index.js]: Rename variables to atelier

**Review by juliebarwick — COMMENTED** [review]
looks good except for atelier spelling @mangohead33

File: server/atlier.js (hunk 1 above)
Comment (mangohead33) [reply server/atlier.js]: Sounds good 👍

**Pull request merged**